akka-stream/src/main/scala/akka/stream/stage/GraphStage.scala

```diff
@@ -55,7 +58,7 @@ abstract class GraphStage[S <: Shape] extends Graph[S, Unit] {
  *    of the enclosing [[GraphStage]]
  *  * Possible mutable state, accessible from the [[InHandler]] and [[OutHandler]] callbacks, but not from anywhere
  *    else (as such access would not be thread-safe)
- *  * The lifecycle hooks [[preStart()]] and [[postStop()]]
+ *  * The lifecycle hooks preStart() and postStop()
```

Comment:
Well, I just followed the "please remove links like these, they don’t work and fail the Java 8 javadoc" sentence. You can interpret the "links like these" in various ways :)